### ISSUE TITLE
ERROR "Converting circular structure to JSON"

### ISSUE THREAD

**Issue by swizardlv** [watson-developer-cloud/speech-to-text-utils#4]
I got this error returned when execute "watson-speech-to-text-utils set-credentials -u  -p"

![image](https://cloud.githubusercontent.com/assets/10117682/21671176/95f2603e-d354-11e6-90a6-e1bd5249a901.png)

**Comment by nfriedly** [comment]
Confirmed - I can reproduce this with the credentials from the Node SDK's integration tests.

```sh
~/language-translator-nodejs (master) $ watson-speech-to-text-utils set-credentials -u username -p password
401 - Not Authorized
~/language-translator-nodejs (master) $ watson-speech-to-text-utils set-credentials -u '[snip]' -p '[snip]'
TypeError: Converting circular structure to JSON
~/language-translator-nodejs (master) $ node -v
v7.1.0
~/language-translator-nodejs (master) $ npm -v
3.10.9
~/language-translator-nodejs (master) $ npm ls -g watson-speech-to-text-utils --depth 0
/Users/nfriedly/.nvm/versions/node/v7.1.0/lib
└── watson-speech-to-text-utils@0.1.2 
```

**Comment by germanattanasio** [comment]
fixed in e0743ffc2743518892b855e8b85d01e7ec45eeb3
